### DISCUSSION TITLE
How can i get PRO badge ?

### DISCUSSION THREAD

**Discussion by HamedAp** [community/community#53661] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi . 
Would you please tell me how to get PRO  badge in my profile ?

**Comment by Ankitz007** [comment, marked as answer]
Hi. There are a few ways I know of, through which you can get the pro badge. 

1. Upgrade to Github Pro. (Go to your profile, `Billing and plans` -> `Plans and usage` -> `Upgrade`)
2. If you are a student, you can avail Github pro using your institution email. (Check [Github Education](https://education.github.com/students) for more information.)
3. If you contribute to open-source on Github, they can offer you pro membership. (Not entirely sure though, please check it out.)

**Reply by Manuel934** [reply]
De hecho con Git Education acabo de verificar y todo funciona de maravilla, tiene un montón de beneficios.
Obviamente en mi segunda cuenta de github: @Manuelillo-dev
<img width="1795" height="918" alt="Captura de pantalla 2025-09-10 134204" src="https://github.com/user-attachments/assets/2a0bd0b1-8a39-463e-b0b1-b046d1f23fea" />

GitHub Student Developer Pack: https://education.github.com/pack

**Comment by Pathum-lakshan** [comment]
cani get

**Comment by Pathum-lakshan** [comment]
you need to member of git hub developer program

**Comment by divyaGrvgithub** [comment]
Hi. There are a few ways I know of, through which you can get the pro badge.

Upgrade to Github Pro. (Go to your profile, Billing and plans -> Plans and usage -> Upgrade)
If you are a student, you can avail Github pro using your institution email. (Check [Github Education](https://education.github.com/students) for more information.)
If you contribute to open-source on Github, they can offer you pro membership. (Not entirely sure though, please check it out.)

**Reply by bsqqq** [reply]
test

**Comment by adityadorge** [comment]
Hello!
There are several methods I am aware of for obtaining the pro badge:

Upgrade to Github Pro: Navigate to your profile, then go to Billing and plans -> Plans and usage -> Upgrade.
If you are a student, you can acquire Github Pro by using your institution email. Visit Github Education for more detailed information.
Contributing to open-source projects on Github might also make you eligible for pro membership. However, I recommend verifying this information for certainty. Please check it out to confirm.

**Comment by bsqqq** [comment]
test

**Comment by arijiiiitttt** [comment]
If you are a student, you can avail Github pro using your institution email. (Check [Github Education](https://education.github.com/students) for more information.) that's better
